src/main/resources/templates/clientvendor/client-vendor-edit.html

```diff
@@ -55,31 +55,31 @@
                                                                 <div class="controls">
                                                                     <label for="users-edit-username">Name</label>
-                                                                    <input type="text" id="users-edit-username"
-                                                                           class="form-control" placeholder="company name" required aria-invalid="false">
+                                                                    <input type="text" id="users-edit-username" th:field="*{companyName}"
+                                                                           class="form-control" placeholder="Company Name" required aria-invalid="false">
                                                                 </div>
                                                             </div>
                                                             <div class="form-group">
                                                                 <div class="controls">
                                                                     <label for="users-edit-email">E-mail</label>
-                                                                    <input type="email" id="users-edit-email"
-                                                                           class="form-control" placeholder="email"
+                                                                    <input type="email" id="users-edit-email" th:field="*{email}"
+                                                                           class="form-control" placeholder="Email"
                                                                            value="email@email.com" required
                                                                            aria-invalid="false">
                                                                 </div>
                                                             </div>
                                                             <div class="form-group">
                                                                 <div class="controls">
                                                                     <label for="users-edit-state">State</label>
-                                                                    <input type="text" id="users-edit-state"
-                                                                           class="form-control" placeholder="etate"
-                                                                           value="state" required
+                                                                    <input type="text" id="users-edit-state" th:field="*{state}"
+                                                                           class="form-control" placeholder="State"
+                                                                           value="Newyork" required
```

Comment:
Looks like in this page there are couple values hardcoded,
do we need to add "TODO" to come back to it later ?